dessia_common/workflow/core.py

```diff
@@ -543,6 +544,17 @@ def copy_pipes(self, copied_workflow: 'Workflow') -> List[Pipe]:
 
         return copied_pipes
 
+    def branch_by_selector(self, blocks: List[Block]):
+        """
+        Return the corresponding branch to each display or export selector
+        """
+        selector_branches = {}
+        for block in blocks:
+            branch = self.secondary_branch_blocks(block)
+            selector = self.block_selectors[block]
+            selector_branches[selector] = branch
```

Comment:
why not just : 
`selector_branches[self.block_selectors[block]] = self.secondary_branch_blocks(block)
`

Reply:
I'll call personal preference on this one. I wouldn't mind if I saw what you propose in a code review (or at least, i wouldn't mind if i didn't think splitting on several lines would greatly increase readability), but i find it easier to read and understand when splitting it as a three-liner while naming the concepts. As the function is not that big, it is not a big deal to have it like this imo